dali/imgcodec/decoders/memory_pool.cc

```diff
@@ -233,6 +234,11 @@ int DeviceNew(void **ptr, size_t size) {
   // this function should not throw, but return a proper result
   try {
     *ptr = GetBuffer<mm::memory_kind::device>(std::this_thread::get_id(), size);
+    if (*ptr) {
```

Comment:
Do we need to erase the memory?

Reply:
No, we don't. It was there just for debugging.